### ISSUE TITLE
Using with WP REST API

### ISSUE THREAD

**Issue by virajsoni06** [carrot/roots-wordpress#13]
Hello,

I came across Roots a couple of days back and it looks pretty great. The site I'll be making has to be on WordPress.

I would like to know if it is possible to use Roots + WordPress without having Jetpack and a WordPress.com site? Since the site will be customized quite a bit on the back-end side of it.

Is it possible to use this plugin with WP REST API?

**Comment by jescalan** [comment]
Yeah, if I'm not mistaken jetpack uses wp rest api behind the scenes anyway. Give it a shot and if something isn't working out, we can tinker with it and make it work. Roots can consume any API, it's just a matter of convenience through the extension options.

**Comment by vivekdhar** [comment]
Not trying to hijack this thread but since I was attempting to do the same thing @virajsoni06 is -- I thought I'd comment here. Any luck?

When using the WP REST API plugin -- what does the `site` need to point to?  I get a `Potentially unhandled rejection [4] [object Object] (WARNING: non-Error used)`.

I've tried a few different things:
`site: 'handelgroup.staging.wpengine.com/wp-json/wp/v2'`
`site: 'handelgroup.staging.wpengine.com`
`site: 'handelgroup.staging.wpengine.com/wp-json/wp/v2/posts'`

@jescalan thoughts? Roots looks pretty great!
